great_expectations_cloud/agent/agent.py

```diff
@@ -231,17 +229,29 @@ def _handle_event_as_thread_exit(
         error = future.exception()
         if error is None:
             result: ActionResult = future.result()
-            status = JobCompleted(
-                success=True,
-                created_resources=result.created_resources,
-            )
-            print(f"Completed job: {event_context.event.type} ({event_context.correlation_id})")
+
+            if result.type == UnknownEvent().type:
+                status = JobCompleted(
+                    success=False,
+                    created_resources=[],
+                    error_stack_trace="The version of the GX Agent you are using does not support this functionality. Please upgrade to latest.",
```

Comment:
Non blocking suggestion: add an error code here e.g. `error_code="agent-unknown-event"`

Reply:
id like to move to another PR